src/app/rundown/components/offsetable-piece/offsetable-piece.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { OffsetablePieceComponent } from './offsetable-piece.component';
+import { Piece } from '../../../core/models/piece'
+import { instance, mock } from '@typestrong/ts-mockito'
+
+describe('PieceComponent', () => {
+  let component: OffsetablePieceComponent;
```

Comment:
Mismatch between component being testet and the string given to the describe, unless i am mistaken, for some reason.

Reply:
Any reason why `OffsetablePieceComponent.name`, is not being used?
This is not the only test file where that isn't used.

Reply:
Yes. This is autogenerated and then the component name has been changed. I'll fix it.

Reply:
Soooo, when's the change comming? :)